### ISSUE TITLE
isForFirstItem has no effect / broken

### ISSUE THREAD

**Issue by josiahbryan** [html-next/vertical-collection#220]
`renderFromLast` / `idForFirstItem` seem to have no effect - I'm recording the last visible ID in the `lastVisiblechanged` handler (I see the output in console), and I set `lastVisibleId` in my controller before `filteredQueryResults` is available, yet the list still starts at the top (very first index), even though I scrolled down the page below the top. Am I missing something obvious here? (The lastVisibleId is in the list when I scroll down to it again, so I know it's there. It may be paged in depending on the list position - e.g. via `lastReached` requesting another page of data). Is this `isForFirstItem` feature even working currently?

(Running version `1.0.0-beta.12` of `@html-next/vertical-collection`)

Here's the template call to the component:

```handlebars
{{#vertical-collection filteredQueryResults
			estimateHeight=50
			staticHeight=false
			containerSelector="body"
			key='id'
			bufferSize=1
			lastReached="loadMore"
			renderFromLast=true
			idForFirstItem=lastVisibleId
			lastVisibleChanged=(action 'lastVisibleChanged')
			as |attraction index|
		}}
```

**Comment by pzuraq** [comment]
You shouldn’t need to do both. VC will currently do one full render before it goes all the way to the bottom so it’s possible that by reading the lastVisibleId and setting it as the idForFirstItem, you are forcing it back to the top.

**Comment by josiahbryan** [comment]
@pzuraq Good point - actually, the `lastVisibleChanged` action **just** stores it in `localStorage`:

```javascript
		lastVisibleChanged(model, index, lastVisibleId) {
			console.log("[lastVisibleChanged]", model.getProperties('name', 'id'), lastVisibleId )
			const storage = window.localStorage;
			if(storage) {
				storage.setItem(this.lastVisibleIdCacheKey(), lastVisibleId);
			}
		},
```

It never actually touches the `lastVisibleId` property on the controller from that action handler - the `lastVisibleId` is **only** set in **one** place in code - it's the part that makes the initial ajax call to load data. It loads that id from local storage, and I can confirm it only does it once because I only see the `console.log` line in console once, not every time I scroll. Here's the relevant code for completeness: 

```javascript
		const storage = window.localStorage;
		if(storage) {
			this.set('lastVisibleId', storage.getItem(this.lastVisibleIdCacheKey()));
			console.log("[initQueryInternal] set lastVisibleId to", this.get('lastVisibleId'));
		}
```

**Comment by pzuraq** [comment]
Interesting, so are you trying to have it scroll to that item as the last visible one? That should be how it works if so. Will try to take a look when I get a chance

**Comment by josiahbryan** [comment]
@pzuraq yes, exactly - I simply want it to show the item given in my `lastVisibleId` as the last one in the page when it renders the list initially - even if that item is in the next "page" of data (e.g. it may only be present in the item set after `lastReached` is called and the server returns more data.)

Let me know if this is a glitch or if my use case is not supported. If my use case isn't supported, I'll try to hack together some sort of forced scroll solution, but that just feels dirty - I'd rather let VC handle it internally if it can.

If you need any other data or info from me, just let me know...
